tutorials/roofit/rf313_paramranges.py

```diff
@@ -22,10 +22,10 @@
 
 # Define 3 dimensional pdf
 z0 = ROOT.RooRealVar("z0", "z0", -0.1, 1)
-px = ROOT.RooPolynomial("px", "px", x, ROOT.RooArgList(ROOT.RooFit.RooConst(0)))
-py = ROOT.RooPolynomial("py", "py", y, ROOT.RooArgList(ROOT.RooFit.RooConst(0)))
-pz = ROOT.RooPolynomial("pz", "pz", z, ROOT.RooArgList(z0))
-pxyz = ROOT.RooProdPdf("pxyz", "pxyz", ROOT.RooArgList(px, py, pz))
+px = ROOT.RooPolynomial("px", "px", x, [0])
+py = ROOT.RooPolynomial("py", "py", y, [0])
```

Comment:
```suggestion
px = ROOT.RooPolynomial("px", "px", x, [0.0])
py = ROOT.RooPolynomial("py", "py", y, [0.0])
```
You have to explicitly pass floating point numbers if you want to use the implicit `RooConst` feature. Otherwise, the zero is interpreted as a `nullptr` and we'll get segfaults.

Thanks for finding this out by the way, by trying to pass the integers! For this PR, you should change the `0` to `0.0` to make it work, but as a next step for another PR we definitely have to find a way to get a nicer error when this `nullptr` confusion happens, or completely avoid the error.